lib/fmrest/spyke/json_parser.rb

```diff
@@ -1,5 +1,7 @@
 # frozen_string_literal: true
 
+require "json"
```

Comment:
I have no idea if this is needed, or if this is the place to put it. Everything works if we leave this line out. Presumably Faraday is loading the JSON library for us (and other gems as well?), but maybe it's not good to depend on other gems to load required libraries?

Reply:
Let's leave it in.